travelwise/frontend/templates/travel/create.html

```diff
@@ -65,13 +97,45 @@ <h2>Activities</h2>
         userMessageDiv.classList.add("user");
         userMessageDiv.innerText = value;
         chatMessages.appendChild(userMessageDiv);
+        
+        for (const message of response.messages) {
+            const div = document.createElement("div");
+            div.classList.add(message.role);
+            div.innerText = message.content;
```

Comment:
Its fine to do this right? No need to clean, since we trust ChatGPT responses.

Reply:
innerText doesn't create new HTML nodes.